doc/adv-manual/general.rst

```diff
@@ -1896,3 +1896,127 @@ the engine version. Here are a few examples::
+
+Showing calculation attributes
+------------------------------
+
+The command `oq show_attrs` offers a convenient way to retrieve
+the attributes of a calculation without needing to open the
+datastore with any external tools::
+
+  $ oq show_attrs -h
+  usage: oq show_attrs [-h] key [calc_id]
+
+  Show the attributes of a HDF5 dataset in the datastore.
+
+  positional arguments:
+    key         key of the datastore
+    calc_id     calculation ID [default: -1]
+
+  options:
+    -h, --help  show this help message and exit
+
+If the key `/` is requested, the root attributes are retrieved,
+i.e. `checksum`, `date` `engine_version` and `input_size`.
+If the calculation id is not specified, the value of the
+requested key is retrieved for the latest calculation.
+
+Mosaic-related commands
+-----------------------
+
+Mosaic-related commands are defined as subcommands of the `oq mosaic` command::
+
+  $ oq mosaic -h
+  usage: oq mosaic [-h] {run_site,sample_rups,sample_gmfs} ...
+
+  positional arguments:
+    {run_site,sample_rups,sample_gmfs}
+                          available subcommands; use oq mosaic <subcmd> --help
+
+  options:
+    -h, --help            show this help message and exit
+
+The `oq mosaic run_site` subcommand gives the possibility to run a PSHA
+analysis for a site on the given longitude and latitude, or for multiple sites
+specified in a CSV file::
+
+  $ oq mosaic run_site -h
+  usage: oq mosaic run_site [-h] [--hc HC] [-s SLOWEST] [-c 8] lonlat_or_fname
+
+  Run a PSHA analysis on the given lon, lat
+
+  positional arguments:
+    lonlat_or_fname       lon,lat of the site to analyze or CSV file
+
+  options:
+    -h, --help            show this help message and exit
+    --hc HC               previous calculation ID
+    -s SLOWEST, --slowest SLOWEST
+                          profile and show the slowest operations
+    -c 8, --concurrent-jobs 8
+                          maximum number of concurrent jobs
+
+If a CSV file is provided, it must contain in each row a site identifier
+starting with the 3-character code of the mosaic model that covers it, and the
+longitude and latitude of the site, separated by commas. In this case, it may
+be convenient also to define environment variables to select or exclude subsets
+of sites from those specified in the CSV file:
+
+* `OQ_ONLY_MODELS`: a comma-separated list of mosaic models (each identified by
+  the corresponding 3-charracters code) to be selected, excluding sites
+  covered by other models
+* `OQ_EXCLUDE_MODELS`: same as above, but selecting sites covered by all models
+  except those specified in this list
+* `OQ_ONLY_SITEIDS`: a comma-separated list of site identifiers to be selected,
+  excluding all others from the analysis
+* `OQ_EXCLUDE_SITEIDS`: a comma-separated list of site identifiers to be excluded,
+  selecting all the others
+* `OQ_MAX_SITES_PER_MODEL`: the maximum quantity of sites to be selected between
+  those covered by each model
+
```

Comment:
I don't like depending on environment variables (IMNSHO they are meant for debugging, NOT for regular usage), however this is another discussion, independent from this PR.

Reply:
Done